### ISSUE TITLE
Kali patches should have a clear diff with only code removal 

### ISSUE THREAD

**Issue by monperrus** [SpoonLabs/nopol#151]
when angelic value and observed value are all false

**Comment by tdurieux** [comment]
why?

**Comment by monperrus** [comment]
because we spend a lot of time to understand that the Dynamoth condition always evaluates to false.

Let me rephrase:

* we don't want any synthesis for Kali patch, in particular the Dynamoth synthesis that yields many of them
* we want Kali-like patches to be output as a diff with only deletion.

**Comment by monperrus** [comment]
if patch is `if (true)` then diff should contain the else block removed.

if patch is `if (false)` then diff should contain the if block removed.
